src/networkxio/gobjfs_server.cpp

```diff
@@ -83,3 +87,27 @@ int gobjfs_xio_server_stop(
 
   return 0;
 }
+
+
+namespace logging = boost::log;
+
+int gobjfs_init_logging(int level){
+  if(level < 0 || level > 5){
+      return 1;
+  };
+  std::vector<boost::log::v2_mt_posix::trivial::severity_level> severities {
```

Comment:
This should be `std::vector<logging::trivial::severity_level>` as the `v2_mt_posix` component depends on boost's configuration.